trinity-mobile-ci/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:8.11.2-stretch
+
+# Install react-native-cli
+RUN yarn global add react-native-cli
```

Comment:
I left [react-native-cli](https://github.com/facebook/react-native/tree/master/react-native-cli) as part of the Dockerfile because it's almost never updated